### ISSUE TITLE
Question on Javascript and CSS Optimizations

### ISSUE THREAD

**Issue by applecrusher** [getgrav/grav#1154]
Hello All,

I am currently using grav for the first time. I installed it and ran a test after installation using Google Page insights, the results are below: 

https://developers.google.com/speed/pagespeed/insights/

<img width="1058" alt="screen shot 2016-11-06 at 8 47 08 pm" src="https://cloud.githubusercontent.com/assets/1963860/20044229/05ab3582-a465-11e6-874b-5445fed42153.png">
<img width="379" alt="screen shot 2016-11-06 at 8 45 58 pm" src="https://cloud.githubusercontent.com/assets/1963860/20044231/07716670-a465-11e6-852e-6f87175e6903.png">

Mobile was a 67 and Desktop was a 84 score.  A similar score was reported for getgrav.com

My questions are:

Are you currently looking to find ways to address issues to remove render-blocking javascript and  optimize CSS delivery since getgrav is meant to be fast? 

Also are these really true issues to worry about?

Thanks!

**Comment by w00fz** [comment]
This is purely a theme decision (which also means the issue should be reopened in the proper [Antimatter](https://github.com/getgrav/grav-theme-antimatter) repository rather than here in Grav core).

That said, Grav itself does not impose where assets are loaded, if you are a theme developer or you customize a theme, you can decide to load the assets in the bottom group of the page rather than in head. (https://learn.getgrav.org/themes/asset-manager#options).

In antimatter we do indeed load some assets in the head and probably we should change that to be in the bottom, however some people don't fully rely on, or necessary trust, these scores generators and still prefer to keep some things in the head.

**Comment by tidiview** [comment]
Hi applecrusher,

I have been facing the result of the same test: I have worked hard to change completely the way things were.
First to understand what the results.
I followed the courses on the UDACITY site about critical path rendering and after that, was able to make the changes so that now test score is between 85 ~ 90.
That said it is the way w00fz wrote, plus lot's of details... because at the end, it's has absolutely no end!

What I can say, is that 85 ~ 90 is an excellent score and that the google test level is high: most of the site I know around are under the result obtain by grav.

Grav and Antimatter are for people that are ready to do things on their own and it requires time; to me, it's my first site building experience, and because of that, I learned a lot.

I'll ask w00fz to forgive me on being long...

Good luck
